src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/windows-linux-hybrid-cluster.mdx

```diff
@@ -1,125 +1,154 @@
-
-
-```yaml
-global:
-  licenseKey: mykey
-  cluster: mycluster
-  lowDataMode: true
- 
-newrelic-infrastructure:
-  enabled: true
-  nodeSelector:
-    kubernetes.io/os: linux
- 
-kube-state-metrics:
-  enabled: true
-  image:
-    tag: v2.10.0
-  nodeSelector:
-    kubernetes.io/os: linux
- 
-nri-kube-events:
-  enabled: true
-  nodeSelector:
-    kubernetes.io/os: linux
- 
-newrelic-logging:
-  enabled: true
-  enableWindows: true
- 
-nri-metadata-injection:
-  enabled: true
-  nodeSelector:
-    kubernetes.io/os: linux
- 
-newrelic-prometheus-agent:
-  enabled: true
-  verboseLog: true
-  config:
-    kubernetes:
-      integrations_filter:
-        enabled: false
-  nodeSelector:
-    kubernetes.io/os: linux
-```
-
-## Install the bundle [#install-bundle]
-
-Run this command to install the bundle:
-
-
-```shell
-helm upgrade --install newrelic-bundle newrelic/nri-bundle --namespace newrelic -f values-newrelic.yaml
-
-```
-
-When Helm returns, all pods should start, including a logging pod on each Windows node. No other New Relic pods on Windows nodes yet.
-
-
-## Installing newrelic-infrastructure preview for Windows nodes [#newrelic-infra]
-
-Create the values file for the `newrelic-infrastructure` v2 chart. For example, `windows-values.yaml`. The example below is for a cluster with Windows 2022 nodegroup or node pool. Adjust the `windowsOsList` section with the correct values following [these instructions](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#windows-install). This applies if you're running 2019 or 2022.
-
-
-```yaml
-global:
-  licenseKey: mykey
-  cluster: mycluster
- 
-enableLinux: false
-enableWindows: true
-windowsOsList:
-  - version: 2022
-    imageTag: 2-windows-ltsc2022-alpha
-    buildNumber: 10.0.20348
-windowsNodeSelector:
-  kubernetes.io/os: windows
-```
-
-Install the v2.7.2 version of the `newrelic-infrastructure` chart. Run this command:
-
-```shell
-helm upgrade --install newrelic-windows newrelic/newrelic-infrastructure --namespace newrelic --version 2.7.2 -f windows-values.yaml
-```
-
-When Helm returns, you should see `newrelic-windows-newrelic-infrastructure` pods on each Windows node.
-
-## Expected results [#expected-results]
-
-Once everything has been set, the New Relic UI should show your Windows nodes in the Kubernetes cluster. You'll see the infrastructure metrics, running pods, and all the related information. 
-
+To use this integration, you should:
+
+* Have a hybrid cluster for Linux and Windows, with the ability to pull container images to the Windows nodes.
+* Create a namespace:
+
+    ```shell
+    helm repo add newrelic https://helm-charts.newrelic.com
+    helm repo update
+    kubectl create namespace newrelic
+    ```
+
+## Install the integration on a hybrid cluster [#install-hybrid]
+
+<Steps>
+    <Step>
+
+### Define your subcharts [#install-chart-bundle]
```

Comment:
wanted to use different verbs for these steps. not sure if we have better titles to use, but wanted to avoid the 'install install install' repetition, since this step does something slightly different. my instinct could be a distinction w/o a difference though

Reply:
Yes, I agree with that.